crates/web5/src/credentials/mod.rs

```diff
@@ -8,6 +8,7 @@ mod josekit;
 mod jwt_payload_vc;
 pub mod presentation_definition;
 mod sign;
+pub mod status_list_credential;
```

Comment:
slight preference for
```suggestion
mod status_list_credential;
pub use status_list_credential::{StatusListCredential, STATUS_LIST_CREDENTIAL_CONTEXT, STATUS_LIST_CREDENTIAL_TYPE, STATUS_LIST_2021, STATUS_LIST_2021_ENTRY};
```